ui/dashboards/src/components/GridLayout/GridLayout.tsx

```diff
@@ -13,17 +13,18 @@
 import { useState } from 'react';
 import { Responsive, WidthProvider } from 'react-grid-layout';
 import { Box, BoxProps, Collapse, GlobalStyles } from '@mui/material';
-import { GridDefinition, GridItemDefinition } from '@perses-dev/core';
+import { GridDefinition } from '@perses-dev/core';
+import { ErrorAlert, ErrorBoundary } from '@perses-dev/components';
 import { styles } from '../../css/styles';
 import { useEditMode } from '../../context';
 import { GridTitle } from './GridTitle';
+import { GridItemContent } from './GridItemContent';
 
 const ResponsiveGridLayout = WidthProvider(Responsive);
 
 export interface GridLayoutProps extends BoxProps {
   groupIndex: number;
   definition: GridDefinition;
-  renderGridItemContent: (definition: GridItemDefinition, itemIndex: number) => React.ReactNode;
```

Comment:
i've been wanting to remove this for so long 😆

Reply:
Hahahaha, that was back when I thought `GridLayout` might be more generic/reusable on its own, but good to just "embrace the store".